maintainer/travis/build_docker.sh

```diff
@@ -17,15 +17,16 @@ if [ -z "$image" ]; then
 fi
 
 if [ "$TRAVIS_OS_NAME" = "linux" ]; then
-	image=espressomd/buildenv-espresso-$image
-	docker run -u espresso --env-file $ENV_FILE -v ${PWD}:/travis -it $image /bin/bash -c "git clone /travis && cd travis && maintainer/travis/build_cmake.sh"
+	image=espressomd/buildenv-espresso-$image:python
```

Comment:
There is no reason why the Docker image should depend on whether we're building master or python. Docker Hub should use the Dockerfile from the python branch to build the default tag (latest), just as before.